runner/output_writer.go

```diff
@@ -29,33 +24,19 @@ func NewOutputWriter() (*OutputWriter, error) {
 	return &OutputWriter{cache: lastPrintedCache}, nil
 }
 
-func (o *OutputWriter) AddWriters(named ...NamedWriter) {
-	o.namedWriters = append(o.namedWriters, named...)
-}
-
-// WriteAll writes the data taken as input using
-// all the writers.
-func (o *OutputWriter) WriteAll(data []byte) {
-	o.Lock()
-	defer o.Unlock()
-
-	for _, w := range o.namedWriters {
-		_, _ = w.Writer.Write(data)
-		_, _ = w.Writer.Write([]byte("\n"))
-	}
+func (o *OutputWriter) AddWriters(named ...io.Writer) {
```

Comment:
Please update the variable name: `named` => `writers`

Reply:
Done